src/app/badges/achievements-list/achievements-list.component.html

```diff
@@ -0,0 +1,27 @@
+<h2 class="title">full list of achievements</h2>
+
+<table mat-table [dataSource]="achievements">
+  <ng-container matColumnDef="title">
+    <th mat-header-cell *matHeaderCellDef> Achievement </th>
+    <td mat-cell *matCellDef="let element">
```

Comment:
these are one-level tags. add empty line between them
and check all files and properly format one-level spaces :)